packages/esm-form-entry-app/src/app/form-submitted/form-submitted.component.html

```diff
@@ -1,20 +1,21 @@
-<div class="omrs-padding-8 omrs-margin-top-8 form-submitted">
-    <div class="success-message">
-        <div>
-            <svg class="omrs-icon" fill="var(--omrs-color-success)" style="height:2rem; width: 2rem;">
-                <use xlink:href="#omrs-icon-check-circle"></use>
-            </svg>
-        </div>
-        <div class="omrs-medium omrs-type-body-large omrs-padding-4">
-            The form was successfully submitted.
-        </div>
-    </div>
-    <div class="omrs-margin-left-8 omrs-margin-top-20">
-        <a class="omrs-link omrs-outlined-action omrs-margin-right-8" (click)="edit()">
-            Edit Form
-        </a>
-        <a class="omrs-link omrs-outlined-action" (click)="cancel()">
-            Close
-        </a>
-    </div>
+<div class="form-submission-container">
+  <div class="success-message">
+      <span>The form was successfully submitted.</span> 
+      <svg class="omrs-icon" fill="var(--omrs-color-success)"
```

Comment:
Should we maybe update this to match other system notifications if possible?

Reply:
@enyachoke i have updated to be consistent with the way other forms are save here is a recording

https://user-images.githubusercontent.com/28008754/148534532-b411ad85-b3d3-410e-ab95-b9132ddd083c.mov